flink-yarn/src/main/java/org/apache/flink/yarn/YarnResourceManager.java

```diff
@@ -105,6 +106,14 @@
 
 	private final YarnConfiguration yarnConfig;
 
+	private static final String RM_FAIR_SCHEDULER = "org.apache.hadoop.yarn.server.resourcemanager.scheduler.fair.FairScheduler";
```

Comment:
Not only `FairScheduler`, but also `SLSFairScheduler` is using the increment allocation configurations.